### PR TITLE
Feature/scoring logic

### PR DESCRIPTION
**Pull request by govTechSatish**
Added Scoring Logic

### CODE REVIEW

File: app/calculation/score-engine.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
+
+const { first } = require('lodash')
+const bandHigh = 'High'
+const bandLow = 'Low'
+const bandMedium = 'Medium'
+
+class ScoreEngine {
+  constructor (desirabilityAssessment) {
+    this.scoringData = require('./score-data.json')// score-data.json
+    this.desirabilityAssessment = desirabilityAssessment// grant-scheme.json
```
Comment (paulsimonandrews) [inline app/calculation/score-engine.js:10]: Remove the comments on these two lines

File: app/calculation/score-engine.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    // remove noShowResult questions
+    this.desirabilityAssessment.desirability.questions = this.desirabilityAssessment.desirability.questions.filter(question => noShowResultQuestions.every(noShow => noShow.key !== question.key))
+
+    // get overall rating band
+    this.desirabilityAssessment.desirability.overallRating.band = GetOverAllRatingBand(bandScore, this.scoringData.desirability)
+    return this.desirabilityAssessment
+  }
+}
+
+function GetOverAllRatingBand (bandScore, sectionScoringData) {
```
Comment (paulsimonandrews) [inline app/calculation/score-engine.js:42]: lowercase beginning of function name 

GetOverAllRatingBand -> getOverAllRatingBand

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
👍 all looks good to me. Just a couple of very minor formatting changes.

File: app/calculation/score-engine.js (hunk 1 above)
Comment (govTechSatish) [reply app/calculation/score-engine.js]: Updated.

File: app/calculation/score-engine.js (hunk 2 above)
Comment (govTechSatish) [reply app/calculation/score-engine.js]: updated.

**Review by paulsimonandrews — APPROVED** [review]

**Pull request merged**